src/main/java/com/laa/model/BankAccount.java

```diff
@@ -54,4 +55,12 @@ public void setBalance(BigDecimal balance) {
 		this.balance = balance;
 	}
 
+	public String getSortCode() {
+		return sortCode;
+	}
+
+	public BankAccount setSortCode(String sortCode) {
+		this.sortCode = sortCode;
+		return this;
```

Comment:
this should follow the typical pojo model, builder pattern should be kept separate